src/client/containers/BoardPageComponents/CandidateList/updateBallots.js

```diff
@@ -0,0 +1,22 @@
+import { useEffect } from 'react';
+
+export async function useUpdateBallots(
+  userId,
+  boardId,
+  candidates,
+  draggedInit,
+) {
+  useEffect(() => {
+    if (!draggedInit) return;
+    (async () => {
+      await fetch(`/api/users/${userId}/boards/${boardId}/ballots`, {
+        method: 'PUT',
+        mode: 'cors',
+        headers: {
+          'Content-type': 'application/json',
+        },
+        body: JSON.stringify(candidates),
+      });
+    })();
+  }, [userId, boardId, candidates, draggedInit]);
```

Comment:
Lint error triggers if I didnt use userId, boardId and draggedInit in the dependency array. Is it ok to put them here?

Reply:
I think its okay for now.